src/serializationHelpers.ts

```diff
@@ -49,6 +49,22 @@ export function encodeWord32(value: number): Buffer {
     return Buffer.from(new Uint8Array(arr));
 }
 
+/**
+ * Encodes a 16 bit unsigned integer to a Buffer using big endian.
+ * @param value a 16 bit integer
+ * @returns big endian serialization of the input
+ */
+export function encodeWord16(value: number): Buffer {
+    if (value > 65535 || value < 0) {
```

Comment:
I guess in `encodeWord16` and `encodeWord32` that we should also check `Number.isInteger()`.

Reply:
I guess we should also add the checks for encodeWord8